template/solr/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Use the curl image to download required Solr cores
```

Comment:
missing license header

Reply:
And metadata

Reply:
This also means that we need to publish this images (and we need to get the agreement from the official image maintainers). I don't see the code or doc to publish this image in the files changed. Maybe I've missed it.